### DISCUSSION TITLE
How to set enterMode to CKEDITOR.ENTER_BR for a Text Datatype?

### DISCUSSION THREAD

**Discussion by hoekiesda** [pimcore/pimcore#12019] [Q&A / Support] [answered]
[Documentation ](https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Objects/Object_Classes/Data_Types/Text_Types.html#page_Editor-Configuration) says: 
"It's possible to pass a custom CKEditor config object to the wysiwyg editor."
```
{
  toolbarGroups : [ { name: 'links' }],
  enterMode: CKEDITOR.CKEDITOR.ENTER_BR,
}
```
That might work in code, but the "Editor Configuration" option in the "Specific Settings" for a WYSIWYG Datatype in the Classes does not accept this JSON snippet. See screenshot below.

Pimcore demo says:
```
Uncaught Error: You're trying to decode an invalid JSON String: {
  toolbarGroups : [ { name: 'links' }],
  enterMode: CKEDITOR.CKEDITOR.ENTER_BR,
}
```

Validate JSON would be:
```
{
	"toolbarGroups": [{
		"name": "links"
	}],
	"enterMode": "CKEDITOR.CKEDITOR.ENTER_BR"
}
```
But I doubt that it means the same thing. Pimcore demo accepts this JSON and the toolbar is indeed altered. But no effect on the `<p></p>` wrapping behaviour of CKEditor, see: https://ckeditor.com/docs/ckeditor4/latest/features/enterkey.html

I tried: `"enterMode": "CKEDITOR.CKEDITOR.ENTER_BR"`, `"enterMode": CKEDITOR.CKEDITOR.ENTER_BR`, `"enterMode": "CKEDITOR.ENTER_BR"`.

Now i ask how to set the enterMode via the Class Definition? Thanks upfront!
![image](https://user-images.githubusercontent.com/2698112/166448527-863385a7-8164-4a72-8955-cfaf009ce5fb.png)

**Comment by hoekiesda** [comment, marked as answer]
```
{
	"enterMode": 2
}
```

**Reply by dolmit-tanel-paaro** [reply]
```
{
	"enterMode": CKEDITOR.ENTER_BR
}
```
